zypp/ZConfig.h

```diff
@@ -496,19 +496,17 @@ namespace zypp
       /**
        * Path where the update items are kept (/var/adm)
        */
-      Pathname update_dataPath() const;
+      static Pathname update_dataPath();
```

Comment:
```suggestion
#if LEGACY(1735)
      Pathname update_dataPath() const;
#else
      static Pathname update_dataPath();
#endif
```

Reply:
And accordingly for the other two and in the .cc file.
So we don't need to break the API immediately.

Reply:
Done